src/components/spreadsheet/utils/config-tables.js

```diff
@@ -57,6 +62,33 @@ const propertiesGetter = (params) => {
     }
 };
 
+const RegulatingTerminalCellRenderer = (params) => {
```

Comment:
```suggestion
const RegulatingTerminalCellRenderer = (params) => {
    const regulatingTerminalConnectableId =
        params.data?.regulatingTerminalConnectableId;
    if (
        params.data?.regulatingTerminalVlId ||
        regulatingTerminalConnectableId
    ) {
        return `${params.data.regulatingTerminalConnectableType} (${regulatingTerminalConnectableId} )`;
    } else {
        return null;
    }
};

```

Reply:
create a const for params.data.regulatingTerminalConnectableId as it used two times
using Template literals is more readable in this case